oracledb-native/build.gradle

```diff
@@ -82,22 +82,21 @@ spotbugsTest {
 
 task validateSpotbugs() {
    doLast {
-       if (spotbugsMain.reports.size() > 0 &&
-               spotbugsMain.reports[0].destination.exists() &&
-               spotbugsMain.reports[0].destination.text.readLines().size() > 0) {
-           spotbugsMain.reports[0].destination?.eachLine {
-               println 'Failure: ' + it
-           }
-           throw new GradleException("Spotbugs rule violations were found.");
-       }
+        if (spotbugsMain.reports.size() > 0 &&
+                spotbugsMain.reports[0].destination.exists() &&
+                spotbugsMain.reports[0].destination.text.readLines().size() > 0) {
+            spotbugsMain.reports[0].destination?.eachLine {
+                println 'Failure: ' + it
+            }
+            throw new GradleException("Spotbugs rule violations were found.");
+        }
```

Comment:
Please check the indentation and formatting.

Reply:
Fixed the indentation issue with 5107ebc6e31adf2a48af0feab1cc16c3732657a1